composer.json

```diff
@@ -1,26 +0,0 @@
-{
-    "name": "phpro/symfony-conventions",
-    "description": "The basic style rules",
-    "type": "library",
-    "license": "propriatary",
-    "minimum-stability": "dev",
-    "prefer-stable": true,
-    "require": {
```

Comment:
Should be actually force all these dependencies and the possible dependency conflicts this gives?

Other options are:

```sh
$ mkdir --parents tools/php-cs-fixer
$ composer require --working-dir=tools/php-cs-fixer friendsofphp/php-cs-fixer
```

Or by using `phive` and add the binaries directly to the package.

The advantage is that the tools are downloaded with the package, without messing with the dependencies. If you want to use another version, you can confgure it in your package.